### DISCUSSION TITLE
How do I center the code outputs in revealjs?

### DISCUSSION THREAD

**Discussion by sigolyori** [quarto-dev/quarto-cli#7933] [Q&A] [answered]
### Description

![image](https://github.com/quarto-dev/quarto-cli/assets/26665873/266f1d81-b950-4db5-a8c8-e42b263436de)

The more proper way looks like centering the whole output. But it is left-aligned in revealjs. Is there any way to fix it? 
Thanks in advance.

### Additional information
Quarto version: 1.4.444
Linux version: Ubuntu 22..04 LTS

**Comment by cderv** [comment, marked as answer]
See previous discussion like this one 

* https://github.com/quarto-dev/quarto-cli/discussions/2951

This is all about CSS here

**Reply by sigolyori** [reply]
Thanks. I need to look for some CSS knowledge.
